PROPOSED_FEATURES.md

```diff
@@ -0,0 +1,287 @@
+This article lists all the new features proposed for revision 3 of DOMOverlays API.
+
+## 1. Allow all elements as children in the source
+
+As a developer, I want to use all available HTML elements as children.
+
+
+```html
+<div data-l10n-id="key1">
+  <ul>
+    <li></li>
+    <li></li>
+    <li></li>
+  </ul>
+</div>
+```
+
+```properties
+key1 = This is a list of errors with
+  <ul>
+    <li>Bullet 1</li>
+    <li>Bullet 2</li>
+    <li>Bullet 3</li>
+  </ul>
+```
+
+Proposed behavior: Allow for the source to include elements that define what
+elements can be used in the translation. Recursively nest into them to allow
+for nested structures.
+
+References:
+* [netError.dtd model cannot be currently ported to Fluent](https://searchfox.org/mozilla-central/rev/2a6f3dde00801374d3b2a704232de54a132af389/browser/locales/en-US/chrome/overrides/netError.dtd#28-35)
+
+
+## 2. Allow text-level child elments in translations
+
+As a localizer, I want to be able to use text-level elements such as `<em>` and `<sup>` in translations even if they are not present in the source.
+
+```html
+<div data-l10n-id="key">
+  Text which doesn't need italics in English.
+</div>
+```
+
+```properties
+key = Text which needs <em>italics</em> in another language.
+```
+
+
+## 3. Do not remove child elements from the DOM
+
+As a developer, I want to be sure that a translation will not remove a child
+element which is important for the functioning of the UI. For instance, if
+a broken localization doesn't include the `<button>`, I still want to be able
+to rely on the button being present in the DOM.
+
+
+```html
+<div data-l10n-id="submit-button">
+  <button id="submit">Submit</button>
+</div>
+```
+
+```properties
+submit = No button here.
+```
+
+Proposed behavior: Child elements from the source are always inserted into the
+translation. If they cannot be matched to anything in the translation, they're
+added at the end.
```

Comment:
To be honest I don't understand this proposition. What's the reasoning for it vs. forcing translators to include all elements (the translators could still decide to dump them at the end as a last measure).